node-project-template/.eslintrc.json

```diff
@@ -1,4 +1,3 @@
 {
-  // TODO: Uncomment this before commiting your new project
-  //"extends": "./node_modules/@usabilla/js-styleguide/eslint-config-usabilla/index.js"
```

Comment:
Made build succeed without having to have this commented out. Woo! .eslintignore ftw